### PR TITLE
More logs in `is_potential_spam` from `dispute-coordinator`

### PR DESCRIPTION
**Pull request by tdimitrov**
Add more logs in `is_potential_spam` revealing why a statement was marked as a spam.

### CODE REVIEW

**Review by sandreim — APPROVED** [review]

File: polkadot/node/core/dispute-coordinator/src/initialized.rs [hunk 1]
```diff
@@ -1351,6 +1351,12 @@ impl Initialized {
 				}
 			}
 			for validator_index in new_state.votes().invalid.keys() {
+				gum::trace!(
```
Comment (eskimor) [inline polkadot/node/core/dispute-coordinator/src/initialized.rs:1354]: Should happen rarely enough to justify "debug"?

File: polkadot/node/core/dispute-coordinator/src/initialized.rs [hunk 2]
```diff
@@ -1375,6 +1381,13 @@ impl Initialized {
 			}
 			for (validator_index, (kind, _sig)) in new_state.votes().valid.raw() {
 				let is_backer = kind.is_backing();
+				gum::trace!(
```
Comment (eskimor) [inline polkadot/node/core/dispute-coordinator/src/initialized.rs:1384]: Same, I think at least debug should be fine here.

**Review by eskimor — APPROVED** [review]

File: polkadot/node/core/dispute-coordinator/src/initialized.rs (hunk 1 above)
Comment (tdimitrov) [reply polkadot/node/core/dispute-coordinator/src/initialized.rs:1354]: Okay, let's bump it to DEBUG

**Review by ordian — APPROVED** [review]

File: polkadot/node/core/dispute-coordinator/src/initialized.rs (hunk 2 above)
Comment (tdimitrov) [reply polkadot/node/core/dispute-coordinator/src/initialized.rs]: Bumped

**Pull request merged**